ddl/ddl_api.go

```diff
@@ -1609,6 +1608,21 @@ func checkTableInfoValidWithStmt(ctx sessionctx.Context, tbInfo *model.TableInfo
 			}
 		}
 	}
+	if tbInfo.DirectPlacementOpts != nil {
```

Comment:
What if `DirectPlacementOpts  == nil` and `tbInfo.PlacementPolicyRef != nil`? Do we need to assign `PlacementPolicyRef.ID`?

Reply:
good catch~ addressed.